src/environmentd/tests/server.rs

```diff
@@ -644,16 +644,6 @@ fn test_storage_usage_collection_interval() {
     info!(%pre_create_storage_usage);
     assert_eq!(pre_create_storage_usage, 0);
 
-    // Test that the storage usage for the table is nonzero after it is created
-    // (there is some overhead even for empty tables). We wait out two storage
-    // collection intervals (here and below) because the next storage collection
-    // may have been concurrent with the previous operation.
-    let timestamp = wait_for_next_collection(&mut client, timestamp);
-    let timestamp = wait_for_next_collection(&mut client, timestamp);
-    let post_create_storage_usage = get_storage_usage(&mut client, &shard_id, timestamp);
-    info!(%post_create_storage_usage);
-    assert!(post_create_storage_usage > 0);
```

Comment:
`post_create_storage_usage` is actually 0 because https://github.com/MaterializeInc/materialize/pull/19498 changed how we calculate storage usage, and a created-but-empty shard is considered 0 bytes for... reasons (https://github.com/MaterializeInc/materialize/blob/ea631f8ac69fc2831401eeeb4a8a3c6a6a4ca189/src/persist-client/src/internal/state_versions.rs#L599C16-L605).

The fact this test passes right now is actually due to a race with GC writing rollups in the background, which we shouldn't rely on.